### PR TITLE
Update db with real data

### PR DESCRIPTION
**Pull request by fp2364**
Updated crank/fixtures/init.json with real data.
Updated tests/test_student.py with more tests.
Updated crank/models.py to refelct new database schema.

### CODE REVIEW

File: crank/models.py [hunk 1]
```diff
@@ -50,11 +50,3 @@ class Rating_id(models.Model):
     class Meta:
         unique_together = (("uni", "course"))
 
-class Rating_Average(models.Model):
```
Comment (peterxie) [inline crank/models.py]: why are we removing this class?

File: models.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
+from django.db import models
```
Comment (peterxie) [inline models.py]: please remove this file

File: tests/test_student.py [hunk 3]
```diff
@@ -4,366 +4,79 @@
...
+v1 = "fp12345"  # valid uni
+v2 = "Alpha"  # valid first or last
+v3 = "Qwerty12345"  # valid password
+v4 = "Qwerty12345"  # valid password
+
 
 class TestAccountCreate(TestCase):
     def setUp(self):
         self.factory = RequestFactory()
-    def testSignUpValid(self):
```
Comment (peterxie) [inline tests/test_student.py]: I don't think you want to be removing all the tests tony wrote? unless you 2 discussed it.

File: tests/test_student.py [hunk 4]
```diff
@@ -4,366 +4,79 @@
...
                      "first_name": v2,
-                     "last_name": "Abcdefghijklmnopqrstuvwxyzxyzxyzxyz",
+                     "last_name": v2,
                      "password1": v3,
                      "password2": v3}
         response = client.post('/signup/', data=form_data)
-        self.assertEqual(response.status_code, f)
-
-    #Password tests
-    def Password_1_1(self):
```
Comment (peterxie) [inline tests/test_student.py]: same as above

File: init.json [hunk 5]
```diff
@@ -0,0 +1 @@
...ourse_listing_table"}}, {"model": "auth.permission", "pk": 31, "fields": {"name": "Can add course_ faculty_ table", "content_type": 11, "codename": "add_course_faculty_table"}}, {"model": "auth.permission", "pk": 32, "fields": {"name": "Can change course_ faculty_ table", "content_type": 11, "codename": "change_course_faculty_table"}}, {"model": "auth.permission", "pk": 33, "fields": {"name": "Can delete course_ faculty_ table", "content_type": 11, "codename": "delete_course_faculty_table"}}, {"model": "auth.user", "pk": 1, "fields": {"password": "pbkdf2_sha256$36000$gOZGnvspeZDd$9u46TLLF0ORF9DhTGXOOsXq+kGzLovBr1te1EGmah5Y=", "last_login": "2017-11-15T02:35:12.672Z", "is_superuser": false, "username": "fp2364", "first_name": "Frank", "last_name": "Panettieri", "email": "fp2364@columbia.edu", "is_staff": false, "is_active": true, "date_joined": "2017-11-15T02:29:01.020Z", "groups": [], "user_permissions": []}}, {"model": "crank.profile", "pk": 1, "fields": {"user": 1, "email_confirmed": true}}, {"model": "crank.rating_id", "pk": 1, "fields": {"uni": 1, "course": 1, "usefulness": 5, "lecture_quality": 4, "overall_quality": 5, "oral_written_tests_helpful": 4, "learned_much_info": 5}}]
```
Comment (peterxie) [inline init.json]: please remove this file

File: models.py (hunk 2 above)
Comment (fp2364) [reply models.py]: reading on how to remove files from a pull request. what's the easiest way?

Comment (peterxie) [reply models.py]: just remove the file and commit that, the pull request should pick up the changes

**Comment by fp2364** [comment]
Pete,

Okay. I updated the db with the complete course/faculty pairings and have
to coordinate with Tony, as he has other changes that need to be in sync
with mine.

Frank

**Pull request closed without merging**